.evergreen.yaml

```diff
@@ -77,24 +77,90 @@ functions:
         script: |
           sudo pacman --refresh --noconfirm -S community/docker
           systemctl start docker.service
+  "teardown_docker":
+    - command: shell.exec
+      params:
+        script: |
+          # Remove all Docker images
+          docker rmi -f $(docker images -a -q) &> /dev/null || true
```

Comment:
I was inspired by the operator on this function. I figured you needed to clean up the Docker environment, is it alright to leave some images?

Reply:
You should ask Anton or Rodrigo (or check git history) to find out why that was added to the Operator build. 
I expect it was because over time we filled the disk on the build hosts. 

Could we do something more fine-grained, which only deletes 
1./ AOSB images
2./ that are > N days old ? 

This seems ok to merge, but a follow up could make improvements like that

Reply:
Happy to leave it for now.